java-build/entrypoint.sh

```diff
@@ -50,13 +50,11 @@ else
   packageVersion="1.0.$productBuildNumber"
```

Comment:
This line is the one that confuses/worries me. Will this push our versions backwards to a v1.0.x even if the version advanced beyond that?

Reply:
Good question - this is the historical functionality - so IF your component has provided a build number environment variable we will assume it is a 1.0.BUILD_NUM release.

My suggestions for the `bdp-java-utils` is to not provide a BUILD_NUM; thus entering the first conditional block where we get the version from the pom.xml file.